setup.py

```diff
@@ -119,12 +119,15 @@ def gen_packages_items():
     return packages
 
 
-# If first not installed install second package
-CHOOSE_INSTALL_REQUIRES = [('opencv-python-headless>=3', 'opencv-python>=3')]
-
 install_requires = parse_requirements()
-for main, secondary in CHOOSE_INSTALL_REQUIRES:
-    install_requires.append(choose_requirement(main, secondary))
+try:
+    # OpenCV installed via conda.
+    import cv2
```

Comment:
We can check the minimum version here.

Reply:
I used RuntimeError but I'm not sure what error should be raised or how it should be handled if incorrect version is installed.

Reply:
It is okay to me.